forgetPass.html

```diff
@@ -16,145 +17,144 @@
-font-family: 'Baloo 2', cursive;
-font-size: 18px;
-box-shadow: 0 0 .2rem .2rem rgb(102, 102, 102);
-
-}
-.container:hover{
-  transform:translateY(.2rem);
-  box-shadow: 0 0 .2rem .2rem rgba(245, 159, 46, 0.671);
-  background: rgba(245, 159, 46, 0.199);
-}
-#votingimage{
-  padding-left: 2rem;
-  padding-top: 4rem;
-}
-@media (min-width:320px) and (max-width:991px){
-    #votingimage{
-      margin-right: 1.2rem;
-    }
-}
+<body>
+    <header>
+        <div class="navbar">
+            <div class="main-heading">
 
 
-  </style>
+                <div class="logo">
+                    <div class="flip">
+                        <div class="flip-inner">
+                            <div class="flip-front">
+                                <a href="#" class="endlogo">
+                                    <img src="./Images/Logo/logo1.jpg" id="header-image">
+                                </a>
+                            </div>
+                            <div class="flip-back">
+                                <a href="#" class="endlogo">
+                                    <img src="./Images/Logo/logo2.jpg" id="header-image">
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
 
-<body>
-    <header>
-    <div class="navbar">
-      <div class="main-heading">
-
-        
-        <div class="logo">
-          <div class="flip">
-            <div class="flip-inner">
-              <div class="flip-front">
-                <a href="#" class="endlogo">
-                  <img src="./Images/Logo/logo1.jpg" id="header-image">
-                </a>
-              </div>
-              <div class="flip-back">
-                <a href="#" class="endlogo">
-                  <img src="./Images/Logo/logo2.jpg" id="header-image">
-                </a>
-              </div>
-            </div>
-          </div>
-         </div>
-          
-        <div class="heading">
-          <img class="emblem"  src="./Images/emblem.png"></img>
-            <p class="hindi-heading" style="font-family: 'Hind', sans-serif; font-weight: bolder;" >राष्ट्रीय मतदान सेवा डेमो पोर्टल</p>
-            <p class="english-heading" style="font-weight: bold;">NATIONAL VOTER'S SERVICES DEMO PORTAL</p>
-        </div>
+                <div class="heading">
+                    <img class="emblem" src="./Images/emblem.png"></img>
+                    <p class="hindi-heading" style="font-family: 'Hind', sans-serif; font-weight: bolder;">राष्ट्रीय मतदान सेवा डेमो पोर्टल</p>
+                    <p class="english-heading" style="font-weight: bold;">NATIONAL VOTER'S SERVICES DEMO PORTAL</p>
+                </div>
 
 
-      </div>
+            </div>
 
 
-      <div>
-        <input type="checkbox" class="checkbox" id="chk" />
-        <label class="label" for="chk">
+            <div>
+                <input type="checkbox" class="checkbox" id="chk" />
```

Comment:
Please revert unnecessary changes

Reply:
Sorry for that.
OK bhaiya I will resolve it.
I am little weak in git commands.